release/tag_repos.sh

```diff
@@ -177,6 +182,11 @@ case "${subcmd}" in
 status)
 	check_versions
 	;;
+pre-release)
+	unset repos[2]
+	runtime_repo="github.com/kata-containers/runtime"
+	export VERSION_TO_CHECK=$(cat "${GOPATH}/src/${runtime_repo}/VERSION") && check_versions
```

Comment:
I think/guess this is reply on the runtime repo having been already checked out to the correct PR - as this script will be called for by the runtime repo code on a runtime repo PR being checked.... maybe a small comment to note that - otherwise it could be a little confusing :-)